Python/Product/Analysis/Interpreter/Ast/AstPythonModule.cs

```diff
@@ -205,5 +209,43 @@ string moduleFullName
 
         public void Imported(IModuleContext context) { }
         public void RemovedFromProject() { }
+
+        private static string TryGetDocFromModuleInitFile(string filePath) {
+            if (string.IsNullOrEmpty(filePath) || !File.Exists(filePath)) {
+                return string.Empty;
+            }
+
+            using (var sr = new StreamReader(filePath)) {
+                string quote = null;
+                while (true) {
+                    var line = sr.ReadLine();
+                    if (line == null) {
+                        break;
+                    }
+                    if (line.Length == 0 || line.StartsWithOrdinal("#")) {
+                        continue;
+                    }
+                    if (line.StartsWithOrdinal("\"\"\"") || line.StartsWithOrdinal("r\"\"\"")) {
+                        quote = "\"\"\"";
+                    } else if (line.StartsWithOrdinal("'''") || line.StartsWithOrdinal("r'''")) {
+                        quote = "'''";
+                    }
+                    break;
+                }
+
+                if (quote != null) {
+                    var sb = new StringBuilder();
+                    while (true) {
+                        var line = sr.ReadLine();
+                        if (line == null || line.EndsWithOrdinal(quote)) {
+                            break;
+                        }
+                        sb.AppendLine(line);
```

Comment:
Do we care about the type of line break that was originally in the file?

Reply:
No